### ISSUE TITLE
Do the photos generated by this project have any copyrights?

### ISSUE THREAD

**Issue by Panprogramista123** [suprdory/spiralator#1]

**Comment by suprdory** [comment]
I don't think so, but honestly I don't know! I'd love to know what you own to do with them though.

**Comment by Panprogramista123** [comment]
I wanted to use the spirograve photo for my YouTube channel profile but I
remembered the copyright

śr., 30 paź 2024, 18:14 użytkownik suprdory ***@***.***>
napisał:
